backend/test/game/dice_handler_test.py

```diff
@@ -0,0 +1,60 @@
+import pytest
+from backend.src.game.dice_handler import DiceHandler
+import backend.src.game.dice as dice
+
+
+@pytest.fixture(autouse=True)
+def dice_handler():
+    dice_handler = DiceHandler()
+    return dice_handler
+
+
+def test_start_turn(dice_handler):
+    for die in dice_handler.dice_values:
+        assert die in dice.DieValue
+
+
+def test_re_roll_one_die(dice_handler):
+    dice_handler.start_turn()
+    start_value = dice_handler.dice_values[0]
+    roll_changed_value = False
+    # since technically possible re-roll returns same roll, run 10 times and quit on successful result
+    for _ in range(10):
+        dice_handler.re_roll_dice(0)
+        if not start_value == dice_handler.dice_values[0]:
+            roll_changed_value = True
+            break
+    assert roll_changed_value
+
+
+def test_re_roll_all_dice(dice_handler):
+    dice_handler.start_turn()
+    start_values = dice_handler.dice_values.copy()
+    dice_handler.re_roll_dice([0, 1, 2, 3, 4, 5])
+    all_elements_are_same = True
+    for i in range(0, len(dice_handler.dice_values)-1):
```

Comment:
1. Mock out 1 set
2. Mock out 2 set
3. Get set 1
4. Call for reroll
5. Assert 2nd set is set 2 you set up